File: Year 2/semester 2/object oriented programming/MyJava/Random.java
class Random
{
	public static void main( String[] args )
	{
		float random = (float) Math.random()  ;
		System.out.println("Random number: " + random );
		
		float multiplied = random * 10;
		System.out.println("Multiplied number: " + multiplied );
		
		int randomInt = (int) Math.ceil( multiplied );
		System.out.println("Random Integer: " + randomInt );
	}
}

File: Year 2/semester 2/Interactive WebDev/Notes/Javascript/6-Working with numbers and strings/random.js

function init()
{
 var panel=document.getElementById("panel");

 var i, rand, temp, str, nums = [];

 for( i=1; i<50; i++){ nums[i] = i ; }


 for( i=1; i<50; i++)
 {
   rand = Math.ceil(Math.random()*49);
   temp = nums[i];
   nums[i] = nums[rand];
   nums[rand]=temp;
 }

 str = "Your Six Lucky Numbers:<br>";
 for( i=1; i<7; i++)
 {
   str += nums[i];
   if( i !== 6) { str += " - "; }
 }

 panel.innerHTML = str;


}
window.onload=init;


